PowerShell/ScubaGear/baselines/aad.md

```diff
@@ -4,11 +4,11 @@
 
```

Comment:
For[ policy group 7 license requirements](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/aad.md#license-requirements-6), I think we're missing the updated language: 

"Policies 7.4 through 7.9 require a Microsoft Entra ID P2 license; however, a third-party Privileged Access Management (PAM) solution may also be used to satisfy the requirements. If a third-party solution is used, then a P2 license is not required for the respective policies."

Reply:
just updated!